vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ExperimentalDesignCard.java

```diff
@@ -15,15 +16,11 @@
  */
 public class ExperimentalDesignCard extends CardLayout {
 
-  public record Experiment(String id, String name, String description) {
-
-  }
-
   public ExperimentalDesignCard(Experiment experiment) {
     Icon myIcon = VaadinIcon.FLASK.create();
     myIcon.addClassNames("mt-s", "mb-s");
-    Span experimentDescription = new Span(experiment.description());
-    addTitle(experiment.name);
+    Span experimentDescription = new Span("some missing description");
```

Comment:
Major NIT: It's better to ask the user to do something than to state that it's missing :) 
```suggestion
    Span experimentDescription = new Span("Describe the experiment");
```

Reply:
this is not the place where the user can do anything 😄 
We are missing a way for the user to enter information so this is always populated by dummy data. Later, the description is not nullable as far as I know.